lib/generators/framework/templates/framework_definition.rb.erb

```diff
@@ -0,0 +1,29 @@
+<%
+  def exports_to(field)
+    value = field['ExportsTo']
+    should_export = value.present? && !value&.match(/[!?]/)
+    ", exports_to: '#{value}'" if should_export
+  end
+-%>
+class Framework
+  class Definition
```

Comment:
not sure about having this as both a namespace and a base class for the definition classes is a good idea. how does Ruby/Rails handle that?

Reply:
Will see if we can go with `Base`.